localization/es.ftl

```diff
@@ -360,3 +360,9 @@ better-navigation-opt-ctrlArrow-name = Atajo Ctrl+Flecha.
 better-navigation-opt-ctrlArrow-desc = Te permite utilizar Ctrl+Flecha y Ctrl+Shift+Flecha para avanzar bloques de texto más rápidamente. De manera similar puedes usar Ctrl+Retroceso para borrar bloques.
 better-navigation-opt-scrollableExpressions-name = Expresiones Desplazables
 better-navigation-opt-scrollableExpressions-desc = Añade una barra de desplazamiento horizontal. Mayormente destinado para facilitar la navegación en móvil.
+
+## Shape Generator
+shape-generator-name = Generador de Formas
+shape-generator-desc = Facilita la creación de equaciones para distinas formas.
```

Comment:
There are two little typos here. I think you meant
```suggestion
shape-generator-desc = Facilita la creación de ecuaciones para distintas formas.
```

Reply:
I'm literally Spanish how could this happen 🫠

Reply:
Don't worry. Spanish is also my mother tongue, and sometimes I write "the" instead of "de" 💀